src/scss/components/button/mixins/_hxButton.scss

```diff
@@ -124,7 +124,7 @@
 }
 
 @mixin hxButton-hover--primary {
-  background-color: $cyan-500;
+  background-color: $cyan-700;
```

Comment:
Please update to the new `v2.0` color variables:

https://github.com/HelixDesignSystem/helix-ui/blob/eb547a4294af4f768294ffef085cfa58136718c1/src/scss/_vars.scss#L1-L118

Reply:
Changes have been done, Please review